[7983] Change apostrophes from straight to curly style (#5043)
DFE-Digital/register-trainee-teachers: docs/uploading_funding_csvs.md

### Gotchas

> 1. `TB_summary_upload_{Month}.csv` are received every 2 months (so don’t expect to see one each time)

Reviewer: ```suggestion
1. `TB_summary_upload_{Month}.csv` are received every 2 months (so do not expect to see one each time)
```
